commands/utility/ch.js

```diff
@@ -15,9 +15,16 @@ module.exports = {
         let chapter = interaction.options.getInteger('chapter');
         let newNick;
 
-        currentNick = currentNick.replace(badRegex, '');
-        newNick = `${currentNick} [ch. ${chapter}]`;
-        await interaction.member.setNickname(newNick);
-        await interaction.reply({ content: `Set you to Chapter ${chapter}!`, ephemeral: true });
+        //check bounds
+        if (-1 < chapter < 2000) {
```

Comment:
this could be reduced to a ternary
`chapter = chapter > 0 ? chapter : 1`

Reply:
🤓